bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -35,226 +35,229 @@ Location,Garden,Outdoor,Garden,Gardens,
+Property,Light,,Light,"Lights, Lighting",Control or status of a light
+Property,LowBattery,,Low Battery,,Low battery indication
+Property,MediaControl,,Media Control,,Audio- or visual- media control
+Property,Mode,,Mode,,Operating mode
+Property,Moisture,,Moisture,,Moisture value
+Property,Motion,,Motion,,Motion detected
+Property,Noise,,Noise,,Noise measurement
+Property,Oil,,Oil,,Oil measurement
+Property,Opening,,Opening,,Opening of a door or window etc.
+Property,OpenLevel,Opening,Open Level,,Open position (analog) of a door or window etc.
+Property,OpenState,Opening,Open State,Open Closed,Open state (digital) of a door or window etc.
+Property,Position,,Position,,Position measurement or control
+Property,GeoLocation,Position,Geo Location,,Geo location coordinate
+Property,Power,,Power,,Power measurement
+Property,Presence,,Presence,,Presence detection
+Property,Pressure,,Pressure,,Fluid pressure measurement
+Property,QualityOfService,,Quality of Service,,Quality of service
+Property,Rain,,Rain,,Rain measurement
+Property,SignalStrength,,Signal Strength,,Signal strength measurement
+Property,RSSI,SignalStrength,RSSI,Received Signal Strength Indication,Received signal strength indication
+Property,Smoke,,Smoke,,Smoke detection
+Property,SoundVolume,,Sound Volume,,Sound volume level
+Property,Speed,,Speed,,Speed measurement or setting
+Property,Tampered,,Tampered,,Tampering detected
+Property,Temperature,,Temperature,Temperatures,Temperature measurement
+Property,Tilt,,Tilt,,Tilt of a blind, shade or shutter
+Property,Timestamp,,Timestamp,,Timestamp of an event
+Property,Ultraviolet,,Ultraviolet,UV,Ultraviolet measurement
+Property,Ventilation,,Ventilation,,Related to ventilation
+Property,Vibration,,Vibration,,Vibration level
+Property,Voltage,,Voltage,,Electric voltage
+Property,Water,,Water,,Water measurement
+Property,Wind,,Wind,,Wind speed or direction
+Equipment,AlarmDevice,,Alarm Device,,Alarm device
+Equipment,AlarmSystem,,Alarm System,Alarm Systems,Alarm system
+Equipment,Application,,Application,"App, Software program",Software program
+Equipment,AudioVisual,,Audio Visual,,Audio visual device
+Equipment,Display,AudioVisual,Display,,Visual display
+Equipment,Projector,Display,Projector,"Projectors, Beamer, Beamers",Projector
+Equipment,Television,Display,Television,"Televisions, TV, TVs",Television
+Equipment,MediaPlayer,AudioVisual,Media Player,,Media player
+Equipment,Receiver,AudioVisual,Receiver,"Receivers, Audio Receiver, Audio Receivers, AV Receiver, AV Receivers",Receiver
+Equipment,Screen,AudioVisual,Screen,Screens,Computer screen
+Equipment,Speaker,AudioVisual,Speaker,Speakers,Loudspeaker
+Equipment,Bed,,Bed,,Bed
+Equipment,Camera,,Camera,Cameras,Camera
+Equipment,CleaningRobot,,Cleaning Robot,"Cleaning Robots, Vacuum robot, Vacuum robots",Cleaning robot
+Equipment,Computer,,Computer,,Computer
+Equipment,ControlDevice,,Control Device,,Control device
+Equipment,Button,ControlDevice,Button,,Push button
```

Comment:
would a touch sensitive button considered a push button, even when there is no actual movement?

Reply:
We need to exclude clothes buttons..